standard/attributes.md

```diff
@@ -598,18 +645,20 @@ The use of conditional methods in an inheritance chain can be confusing. Calls m
 > // File `class1.cs`
 > using System;
 > using System.Diagnostics;
+>
 > class Class1
 > {
 >     [Conditional("DEBUG")]
->     public virtual void M() {
+>     public virtual void M() =>
```

Comment:
How would you feel about unwrapping the body of this onto the same line?

Reply:
That fits!